andela/ah-frontend-space
# #164691174 Setup React Router DOM

#### What does this PR do?
- allows usage of routes to switch to different pages.
- allow homepage to be rendered using route ```http://localhost:8080/``` 
- allow login page to be rendered via route ```http://localhost:8080/login```
- add fall back not found page in case route isnt found
#### Description of Task to be completed?
- adding configuration to the webpack.config file to allow the file path for creating builds to be defined and found on the server running.
- adding components like Home and login to be rendered on different login routes.
#### How should this be manually tested?
- clone this repo and run ``` npm install``` for dependencies.
- run ```npm start``` in your terminal or ```http://localhost:8080/``` and your response should be;
<img width="637" alt="Screenshot 2019-07-03 at 11 17 53" src="https://user-images.githubusercontent.com/43015966/60575454-3d3a8500-9d84-11e9-8609-05e90fd8859e.png">

- you can click login link on the page for a redirect or type ```http://localhost:8080/login``` into your browser and you should see
<img width="732" alt="Screenshot 2019-07-03 at 11 21 16" src="https://user-images.githubusercontent.com/43015966/60575706-b3d78280-9d84-11e9-9cca-1fc40fe82145.png">

#### What are the relevant pivotal tracker stories?
[#164691174](https://www.pivotaltracker.com/story/show/164691174)
